packages/action-button/test/action-button-memory.test.ts

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { fixture } from '@open-wc/testing';
+import { m as BlackActionButton } from '../stories/action-button-black.stories.js';
+import { ActionButton } from '@spectrum-web-components/action-button';
+import { testForMemoryLeaks } from '../../../test/testing-helpers.js';
+
+testForMemoryLeaks(
+    async () =>
+        await fixture<ActionButton>(BlackActionButton(BlackActionButton.args))
+);
```

Comment:
Not sure if this is useful or not, but should we signature of this method so it works along the lines of:

```suggestion
testForMemoryLeaks<ActionButton>(BlackActionButton(BlackActionButton.args));
```

I'm not 100% sure the type is even technically required as we're not testing against its actual API here. 

Maybe I'm being overly precious and it's not worrying about, but I'm less excited by the ones blow out to the following by the linter:

```
testForMemoryLeaks(
    async () =>
        await fixture<Underlay>(
            html`
                <sp-underlay></sp-underlay>
            `
        )
);
```

With this change, it would technically be the below:

```
testForMemoryLeaks<Underlay>(html`<sp-underlay></sp-underlay>`);
```

Or even less is the type is unneeded.

🤔

Reply:
I think to me it does improve the readability and conciseness of the test calls.

Reply:
with the current implementation of testForMemoryLeaks adding a type will expect the function to return a value. How do we see if we just update the above to
```
testForMemoryLeaks(html`<sp-underlay></sp-underlay>`);
```